backend-rust/src/graphql_api.rs

```diff
@@ -0,0 +1,4222 @@
+//! TODO
```

Comment:
This file is way too long. It should probably be split into smaller modules. (is it generated?)

Reply:
I don't think the line count alone should dictate this, there needs to be something gained from it.
In any case, I would prefer such things were postponed until the end of the first working solution, since we don't really have the full picture yet.

Reply:
At this scale, I disagree. I think there is very rarely a reason to have a file of several thousand lines and the existence of such a file would have to be justified, rather than having to justify splitting it up.

Reply:
Let's agree to disagree then.